CHANGELOG.md

```diff
@@ -1,19 +1,81 @@
 # Change Log
 This repository contains the `FreeRTOS AWS Reference Integrations`, which are pre-integrated FreeRTOS projects that demonstrate connectivity with AWS IoT. The repository contains projects for many different microcontroller evaluation boards.
 
-## Changes since previous release
+## 202107.00 July 2021
 
```

Comment:
I think we should summarize the new features we're adding in this release. Otherwise, as a user, I'm confused when we say New Features --> OTA, when OTA had already existed in this repo. The summary can be something like, "This release includes the new managed AWS IoT Over-the-Air update (OTA) library, AWS IoT Jobs library, and the AWS IoT Device Defender custom metrics feature from the FreeRTOS 202012.01 LTS release, and the coreMQTT Agent library from the FreeRTOS 202104.00 release."